Makefile

```diff
@@ -0,0 +1,17 @@
+clean:
+	@rm -rf `find . -name __pycache__`
+	@rm -f `find . -type f -name '*.py[co]' `
+	@make -C docs clean
+
+doc:
+	@make -C docs html SPHINXOPTS="-W -E"
+	@echo "open file://`pwd`/docs/_build/html/index.html"
+
+doc-spelling:
+	@make -C docs spelling SPHINXOPTS="-W -E"
```

Comment:
May be move these options to sphinx Makefile?

Reply:
No. 
Sphinx Makefile is autogenerated, I don't want to touch it.